tests/test-animation-panel-handler.js

```diff
@@ -118,7 +118,35 @@ require([
         },
     });
 
-    test("startOptions", function () {
+    test("setOnAnimationStarted", function (assert) {
+        assert.expect(2);
```

Comment:
(just making this clear for folks unfamiliar with QUnit, aka me)
```suggestion
        // Exactly two assertions should be run within this test
        assert.expect(2);
```